src/components/MarcusFEMentor/CountryPage/BackButton.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { useHistory } from 'react-router'
+import { ArrowLeftOutlined } from '@ant-design/icons'
+
+import { PATHS } from '../../../routes/PATHS'
+import { StyledBackButton } from '../styles/CountryPage.styled'
+
+const BackButton = () => {
+  const history = useHistory()
+  return (
+    <StyledBackButton type="default" icon={<ArrowLeftOutlined />} onClick={() => history.push(PATHS.MARCUS_FE_MENTOR)}>
```

Comment:
doesn't work tho, cos if i click on the border countries links, the back button would go to the previous country instead of the main page. so have to push exact path